.docker/Dockerfile

```
@@ -14,7 +14,7 @@
 
 # The spark version should follow the spark version in databricks.
 # The databricks version of spark is controlled from dh3-infrastructure and uses latest LTS (ATTOW - Spark v3.3.0)
-FROM jupyter/pyspark-notebook:spark-3.3.0
+FROM quay.io/jupyter/pyspark-notebook:spark-3.5.0
```

```suggestion
FROM jupyter/pyspark-notebook:spark-3.5.0
```
try this instead, its from [here](https://hub.docker.com/layers/jupyter/pyspark-notebook/spark-3.5.0/images/sha256-f281e57761669b518e8b9e25af53d99b8ee508557a5de59764f1199120303a84?context=explore)

---

In overview of https://hub.docker.com/r/jupyter/pyspark-notebook it says 
Images hosted on Docker Hub are no longer updated. Please, use [quay.io image](https://quay.io/repository/jupyter/pyspark-notebook), hence the change